harness_runner/setup.py

```diff
@@ -3,5 +3,6 @@
 setup(
     name='harness-runner',
     version='0.0.1',
+    py_modules=['util', 'harness_runner', 'ksql'],
```

Comment:
I forgot about this... When I run step 5 of the README setup [instructions](https://github.com/confluentinc/kafka-tutorials) without this, it failed and this was my fix:
```
dtroiano@C02DR18FMD6T harness_runner % pip3 install -e .
DEPRECATION: Configuring installation scheme with distutils config files is deprecated and will no longer work in the near future. If you are using a Homebrew or Linuxbrew Python, please see discussion at https://github.com/Homebrew/homebrew-core/issues/76621
Obtaining file:///Users/dtroiano/repos/kafka-tutorials/harness_runner
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [14 lines of output]
      error: Multiple top-level modules discovered in a flat-layout: ['util', 'harness_runner', 'ksql'].

      To avoid accidental inclusion of unwanted files or directories,
      setuptools will not proceed with this build.

      If you are trying to create a single distribution with multiple modules
      on purpose, you should not rely on automatic discovery.
      Instead, consider the following options:

      1. set up custom discovery (`find` directive with `include` or `exclude`)
      2. use a `src-layout`
      3. explicitly set `py_modules` or `packages` with a list of names

      To find more information, look for "package discovery" on setuptools docs.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.
```

Reply:
Ah, I get it, thanks for adding that.  
We'll probably need to update the README instructions in a separate PR